backend/git_real/tests/test_activity_log_creators.py

```diff
@@ -72,3 +72,30 @@ def test_adding_a_pr_review(self, has_permission):
         self.assertEqual(entry.object_2, repo)
         self.assertEqual(entry.event_type.name, creators.PR_REVIEWED)
 
+
+class log_push_event_Tests(APITestCase):
+    def test_that_timestamp_properly_set(self):
+        push = PushFactory()
+        creators.log_push_event(push)
+        self.assertEqual(LogEntry.objects.first().timestamp, push.pushed_at_time)
+
+    @mock.patch.object(IsWebhookSignatureOk, "has_permission")
+    def test_pushing(self, has_permission):
+        has_permission.return_value = True
+
+        self.assertEqual(Push.objects.all().count(), 0)     
+
+        push = PushFactory()
+        creators.log_push_event(push)
```

Comment:
You still need to test that this function gets called when it is supposed to. Right now the only thing that calls `log_push_event` is the unit tests. But it needs to be called during normal code execution. 

Push instances get created by the github webhook. It's a view in the git_real app. when a push happens on gihub, then github sends an api call to our webhook view. Then we make a Push instance in our db.

It would make sense for the webhook view to cause `log_push_event` to be called.

Reply:
This is very important. Please:

1. Write a test that shows that when a Push is created then it is logged
2. Make the test pass

I'm not sure why this comment was not addressed.  If this is not addressed then the code will not be useful.